### DISCUSSION TITLE
100 stars :star2:

### DISCUSSION THREAD

**Discussion by HugoFara** [HugoFara/lwt#131] [Announcements]
Hello everyone our project finally reached 100 stars! I'm so proud that so many people liked it! Thank you a lot for all your support, bug tracking, pull requests, and all the other kinds of contributions I received!

It has been a lot of efforts to continue on with the wonderful software of lang-learn-guy, and I have learned many things along the way! I hope many good things will continue to happen and that we will all reach our language targets!

**Comment by Optophonic** [comment]
I used to use the 'original' LWT and upgraded to your fork. It's really nice to see the progress on the software and the improvements you made. Thank you for your effort and the great work you did!

**Comment by HugoFara** [comment]
By the way, reaching 100 stars allowed me to apply for OpenCollective!

Honestly, I do not intent to make much money, but if it enough to rent a small server, I will be happy :smile:
